gaia-header.js

```diff
@@ -278,9 +278,9 @@ module.exports = component.register('gaia-header', {
   setTitleStyle: function(el, style) {
     debug('set title style', style);
     this.observerStop();
-    el.style.MozMarginStart = style.marginStart + 'px';
-    el.style.MozPaddingStart = style.padding.start + 'px';
-    el.style.MozPaddingEnd = style.padding.end + 'px';
+    el.style.marginLeft = style.marginStart + 'px';
+    el.style.paddingLeft = style.padding.start + 'px';
+    el.style.paddingRight = style.padding.end + 'px';
```

Comment:
We need `MozMarginStart` for our header to be fully RTL don't we? How would we align the `<h1>` otherwise? Would we have to fallback to padding based solution?